src/Auth0.php

```diff
@@ -182,6 +190,13 @@ class Auth0
      */
     protected $guzzleOptions;
 
+    /**
+     * Skip the /userinfo endpoint call and use the ID token.
+     *
+     * @var boolean
+     */
+    protected $skipUserinfo;
```

Comment:
Config option to skip the userinfo call (helpful if access_token is not scoped for /userinfo or to skip a potentially unnecessary HTTP call).